### ISSUE TITLE
Brazilian portuguese localization

### ISSUE THREAD

**Issue by ghost** [mimoguz/tripeaks-gdx#7]
Hi, I'm a native speaker of brazilian portuguese and would like to help with the translation of the game.
Here is the resource, translated:

```
start=Iniciar
fromStack=Você {0,choice,0#não pegou nenhuma carta|1#pegou uma carta|1<pegou {0,number,integer} cartas} do baralho.
usedUndo=Você {0,choice,0#não desfez nenhum movimento|1#usou Desfazer uma vez|2#usou Desfazer duas vezes|2<usou Desfazer {0,number,integer} vezes}.
longestChain=Sua maior sequência foi de {0,choice,0#zero cartas|1#uma carta|1<{0,number,integer} cartas}.
newGame=Iniciar nova partida
newGameShort=Nova partida
exit=Sair do jogo
return=Voltar à partida
darkTheme=Usar tema escuro
showAll=Mostrar todas as cartas
won=Você venceu!
```

**Comment by mimoguz** [comment]
Hi. Thank you!

Though, I hope you don't mind if I delay this a few days, since I've got the virus :( . I'm quite OK, but want to take it easy for a while.

**Comment by ghost** [comment]
No hurry, I hope you get better! Really enjoying your game, let me know if you need any more help

**Comment by mimoguz** [comment]
Hi! If you are still interested, can you check the test image?

[Image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/i8nTestImages/pt_BR_01.png)

**Comment by ghost** [comment]
Hello! The words, accents and plurals shown in the image are all correct 👍

**Comment by mimoguz** [comment]
Great! And if you don't mind, a few more lines for the store: 

* A tri peaks solitaire game. 
* An open source tri peaks solitaire game.
* Currently only has randomly generated games.
* Pre-solved games option is on the roadmap.

Also the button names on [this image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/main/fastlane/metadata/android/en-US/images/phoneScreenshots/1.png):
* Deal
* Undo
* Menu

And a question: the game is in the Play Store too. Free, without ads or anything, and store page links to this repo. Is it OK for you?

**Comment by ghost** [comment]
- Um jogo de paciência tri peaks.
- Um jogo de paciência tri peaks em código aberto.
- Atualmente possui apenas partidas geradas aleatoriamente.
- A opção de jogar partidas pré-resolvidas está planejada.

<br>

- Pegar uma carta
- Desfazer
- Menu

No problem regarding the Play Store! I'm ok with it

**Comment by mimoguz** [comment]
Released. Thanks!

**Comment by mimoguz** [comment]
Hi.

I'm readying a new release with a few new features, so I need new strings. I appreciate your help on this.

## New strings

* No more moves! ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png), displayed when the game is stalled)
* Undo last move ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png))
* Save ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
* Cancel  ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png)) 
* Start with an empty discard ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png), [result](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/empty_discard.png))
* New game layout: ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
* Options ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_menu.png)

### Layout options
* Three peaks
* Two peaks and a funnel ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_inverted_2nd_layout.png))
* Diamonds ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_diamonds_layout.png))

### Game description
* An open-source tri peaks solitaire game with multiple layout options.

Thanks!

**Comment by ghost** [comment]
Nice additions to the game! Here is the translation:

---
- Sem mais movimentos!
- Desfazer último movimento
- Salvar
- Cancelar
- Iniciar com o descarte vazio
- Estilo de novas partidas
- Opções

Layout options (Estilos de partida)

- Três picos
- Dois picos e um funil
- Diamantes

Game description (Descrição do jogo)

- Um jogo de paciência tri peaks em código aberto, com diversos estilos de partida.

**Comment by mimoguz** [comment]
👍

**Comment by mimoguz** [comment]
Included in new release

**Comment by mimoguz** [comment]
Hello again.

I want to make some small changes for the next release. Could you have look at these:

* Exit   (Before: "Exit **game**")
* Start with an empty discard **pile**   (Before: "Start with an empty discard")
* Two peaks and a **valley**  (Before: "Two peaks and a **funnel**")
* Layout for new games   (Before: "New game layout")
* Your longest chain **had** ...  (Before: "Your longest chain **was** ...")

They may be  already OK on your version though, some (most) of them are because of my rather poor English. 🙂

Thanks.

**Comment by ghost** [comment]
- Sair
- Iniciar com a pilha de descartes vazia
- Dois picos e um vale
- Estilo para novas partidas (this solves the ambiguity in "new game layout"!)
- Sua maior sequência teve ...

Let me know if some of the strings are too long. I came up with shorter translations for the _usedUndo_ and _fromStack_ strings:

```
usedUndo=Você {0,choice,0#não desfez nenhuma jogada|1#desfez uma jogada|2#desfez duas jogadas|2<desfez {0,number,integer} jogadas}.

fromStack=Você {0,choice,0#não pegou cartas|1#pegou uma carta|1<pegou {0,number,integer} cartas} do baralho.
```

**Comment by mimoguz** [comment]
> Let me know if some of the strings are too long. I came up with shorter translations for the usedUndo and fromStack strings:

The current strings are short enough. Are the new ones better/clearer?

**Comment by ghost** [comment]
Yes. They sound more natural and closer to card games terminology

**Comment by mimoguz** [comment]
Included in the release 0.51.

**Comment by mimoguz** [comment]
Hi. I want to add one last thing before calling this 1.0. Could you please check the [issue #14](https://github.com/mimoguz/tripeaks-gdx/issues/14)? Thanks.

**Comment by mimoguz** [comment]
[New strings](https://github.com/mimoguz/tripeaks-gdx/issues/27)
